plenum/test/view_change/test_no_propagate_request_on_different_last_ordered_before_vc.py

```diff
@@ -104,27 +104,27 @@ def test_no_propagate_request_on_different_prepares_on_backup_before_vc(looper,
     primary = getPrimaryReplica(txnPoolNodeSet, slow_instance).node
     non_primaries = [n for n in txnPoolNodeSet if n is not primary]
 
-    check_last_ordered(non_primaries,
-                       slow_instance,
-                       (old_view_no, old_last_ordered[1] + 1))
+    looper.run(eventually(check_last_ordered, non_primaries,
+                          slow_instance,
+                          (old_view_no, old_last_ordered[1] + 1)))
 
     # Backup primary replica must not advance last_ordered_3pc
     # up to the master's value
-    check_last_ordered([primary],
-                       slow_instance,
-                       (old_view_no, old_last_ordered[1]))
+    looper.run(eventually(check_last_ordered, [primary],
```

Comment:
Why a 3PC with new primaries is not sent here (that is last_ordered is not (old_view_no+1, 1))

Reply:
fixed